README.md

```diff
@@ -1,16 +1,38 @@
 # Kymactl
 
+## Overview
+
 A command line tool to support Kyma developers
 
 ## Available Commands
 
-Some of the following commands are still work in progress and are not yet usable
-
-- `version`: Shows the Kymactl version. The version is set at compile time passing it to the go linker as a flag:
+- `version`: Shows the kymactl version. The version is set at compile time passing it to the go linker as a flag:
 
     ```bash
     go build -o kymactl -ldflags "-X github.com/kyma-incubator/kymactl/cmd.Version=1.5.0"
     ```
 - `status`: Tracks the status of a Kyma cluster (replaces the `is-installed.sh` script)
-- `ctx`: Allows managing Kyma contexts (WIP).
-- `help`: Displays usage for the given command (e.g. `kymactl help`, `kymactl help ctx`, etc...)
+- `help`: Displays usage for the given command (e.g. `kymactl help`, `kymactl help status`, etc...)
+
+## kymactl as a kubectl plugin
+
+To follow this section a kubectl version of 1.12.0 or later is recommended.
+
+A plugin is nothing more than a standalone executable file, whose name begins with kubectl- . To install a plugin, simply move this executable file to anywhere on your PATH.
+
+Rename a `kymactl` binary to `kubectl-kymactl` and place it anywhere in your PATH:
+
+```bash
+sudo mv ./kubectl-kymactl /usr/local/bin
```

Comment:
When the user downloads the CLI, the file will be called kymactl probably. So the move should assume that naming. Furthermore, I suggest to name the command with "kyma" only

```suggestion
sudo mv ./kymactl /usr/local/bin/kubectl-kyma
```